test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -247,9 +251,16 @@ spec =
 
             methods.sort . should_equal ['bar', 'baz', 'first_method', 'foo', 'my_method', 'other_method', 'second_method']
 
+        Test.specify "static methods of MyType" <|
+            methods = Meta.meta (Meta.type_of My_Type) . methods
+            methods.sort . should_equal ['Value', 'create', 'factory', 'first_method', 'my_method', 'other_method', 'second_method']
```

Comment:
Interesting. When we ask for _static methods_ on `MyType` we get:
- the `Value` constructor
- static `create` method
- static extension `factory` method
- all the instance methods (as they can also be invoked as static ones)
-  there are _no getters_ - `bar`, `baz`, `foo` cannot be invoked with static syntax!

CCing @hubertp

Reply:
OK for now. Needs a separate ticket, if it shall be fixed.